tools/developer-tools-for-vscode.adoc

```diff
@@ -1,17 +1,117 @@
 = IBM BAMOE Developer Tools for VS Code
 include::../partials/attributes.adoc[]
 
-{PRODUCT_SHORT} provides a handy VS Code Extension that bundles graphical Editors for BPMN and DMN files.
-The {PRODUCT_SHORT} Developer Tools extension can be downloaded from {VS_CODE_BUNDLE_LINK}[*{VS_CODE_BUNDLE}*].
+{PRODUCT_SHORT} provides a handy VS Code Extension that bundles graphical Editors for BPMN, DMN files, and SCESIM files.
+The {DEV_TOOLS} extension can be downloaded from the https://marketplace.visualstudio.com/items?itemName=IBM.bamoe-developer-tools[VS Code Marketplace].
 
-The bundled BPMN and DMN Editors are made available through the KIE DMN Editor and KIE BPMN Editor extensions, and they feature all available features you have on {CANVAS}.
-Static file validation, DMN Runner, and Dev deployments are not available for {PRODUCT_SHORT} Developer Tools yet, but you can achieve a similar experience of DMN Runner with our Scenario Simulation Editor.
+The bundled BPMN, DMN and Test Scenario Editors are made available through the KIE DMN Editor and KIE BPMN Editor extensions, and they feature all available features you have on {CANVAS}.
+Static file validation, DMN Runner, and Dev deployments are not available for {DEV_TOOLS} yet, but you can achieve a similar experience of DMN Runner with our Test Scenario Editor.
 See how below.
 
-== Running unit tests with Scenario Simulation files
+== Running unit tests with Test Scenario
 
-Running Scenario Simulation (.scesim) files requires a configured Java project with the JUnit dependency available.
-Please follow this https://blog.kie.org/2021/04/how-to-use-test-scenario-editor-to-test-your-dmn-asset-in-vscode.html[blog post tutorial] to get started with Scenario Simulation on {PRODUCT_SHORT} Developer Tools.
+=== Overview
+This documentation provides a step-by-step guide on how to use the Test Scenario editor in VSCode to test DMN assets created with Kogito Tooling. For that scope, this guide will refer to an existing project that lives in the https://github.com/apache/incubator-kie-kogito-examples/tree/stable/kogito-quarkus-examples/dmn-pmml-quarkus-example[kie-kogito-examples] GitHub repo. However, this guide will provide the steps to enable Test Scenario in your own project
```

Comment:
Is Kogito Tooling okay here, or should we have something IBM related, e.g. "created with IBM BAMOE..."

Reply:
@kbowers-ibm Thank you, change applied.